.code-samples.meilisearch.yaml

```diff
@@ -4,76 +4,323 @@
 # You can read more on https://github.com/meilisearch/documentation/tree/master/.vuepress/code-samples
 ---
 get_one_index_1: |-
+  client.index('movies')
 list_all_indexes_1: |-
+  client.indexes
 create_an_index_1: |-
+  client.create_index('movies', primaryKey: 'movie_id')
 update_an_index_1: |-
+  client.index('movies').update(primaryKey: 'movie_id')
 delete_an_index_1: |-
+  client.delete_index('movies')
 get_one_document_1: |-
+  index.get_document(25684)
+  // OR
```

Comment:
```suggestion
```

I would rather remove this part and keep only the more "ruby-friendly" way to do it. Or if you absolutely want both, keep the most "ruby-friendly" way at first 😉